include/stx/format_charconv_impl.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+namespace stx::format_impl
+{
+
+/**
+ * Parse an base-10 integer from range `begin` to `end`.
+ *
+ * Returns the iterator to the first non matching character, or `end`.
+ */
+template <class _Iter, class _Int>
+std::pair<_Iter, bool> from_chars(_Iter begin, _Iter end, _Int& out)
```

Comment:
As far as I can tell, we only need this function to parse positive integers (width/index), right? Maybe we can rename it to `parse_unsigned` or so and remove the signed parsing functionality.